#!/usr/bin/python3

def raise_exception_msg(message=""):
    """"
    raises a nameexception with the message
    """
    raise NameError(message)
